src/main/java/mnsky/Parser.java

```diff
@@ -0,0 +1,190 @@
+package mnsky;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import mnsky.exceptions.MnskyException;
+import mnsky.exceptions.MnskyMissingParameterException;
+
+public class Parser {
```

Comment:
Perhaps you could add a header comment to this class? 🛩️ 
I noticed this in some other classes as well